lib/config.js

```diff
@@ -1,6 +1,12 @@
 const fp = require('fastify-plugin');
 
 module.exports = fp(
+  /**
+   * Initialize config
+   * @param {import('fastify').FastifyInstance} fastify
```

Comment:
eslint does not understand `import` calls in JSDoc. You can work around by using a `typedef`:

```js
/**
 * @typedef {import('fastify').FastifyPluginOptions} FastifyPluginOptions
 */
/**
 * @param {FastifyPluginOptions} opts
 */
```